### PR TITLE
Feat/revoke badges

### PR DESCRIPTION
**Pull request by jona159**

### CODE REVIEW

**Review by YouQam — CHANGES_REQUESTED** [review]
When there are two or more recipients, revoking a badge doesn't refresh list of recipients. It refreshes only when there's one recipient.

**Review by YouQam — CHANGES_REQUESTED** [review]
View ≤ 1278px needs to be updated.
Screenshot 2024-07-05 at 17 24 59

**Comment by jona159** [comment]
> View ≤ 1278px needs to be updated. Screenshot 2024-07-05 at 17 24 59

@YouQam The double row layout is on purpose, but for some reason the borders did not get applied (see issuer detail datatable, its the same there). I fixed the borders now

File: src/app/issuer/components/badgeclass-detail/badgeclass-detail.component.ts [hunk 1]
```diff
@@ -184,22 +174,24 @@ export class BadgeClassDetailComponent extends BaseAuthenticatedRoutableComponen
 	revokeInstance(instance: BadgeInstance) {
 		this.confirmDialog
 			.openResolveRejectDialog({
-				dialogTitle: 'Warning',
-				dialogBody: `Are you sure you want to revoke <strong>${this.badgeClass.name}</strong> from <strong>${instance.recipientIdentifier}</strong>?`,
-				resolveButtonLabel: 'Revoke Badge',
-				rejectButtonLabel: 'Cancel',
+				dialogTitle: 'Warnung',
```
Comment (timber-they) [inline src/app/issuer/components/badgeclass-detail/badgeclass-detail.component.ts:177]: Would it be possible to have those strings in the language string file thingies? Then we don't have to hardcode every possible language, should we at some point also support English or other languages.

**Review by timber-they — DISMISSED** [review]

**Comment by YouQam** [comment]
> > View ≤ 1278px needs to be updated. Screenshot 2024-07-05 at 17 24 59
> 
> @YouQam The double row layout is on purpose, but for some reason the borders did not get applied (see issuer detail datatable, its the same there). I fixed the borders now

OK, then update the mobile view as the date of `awarded in` goes to middle where it should be at
Screenshot 2024-07-09 at 13 19 29
 right side. See attached screenshot.

**Review by YouQam — APPROVED** [review]
I approved this PR, **please merge it after:**
- updating mobile view.
- adding translation as suggested by Timber.

**Pull request merged**